tests/cmsis_rtos_v1/prj_cov.conf

```diff
@@ -0,0 +1,6 @@
+CONFIG_ZTEST=y
```

Comment:
This commit is a no-go. You are duplicating lots of Kconfigs for coverage support only for the purpose of increasing stacks. This will go out of sync immediately and is an ugly workaround. All of this commit needs to go away and we need to think how we should do this better. This just does not scale and IMO not immediately necessary